### PR TITLE
fixed duration filtering bug

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: functions/zoom-downloader.py [hunk 1]
```diff
@@ -188,10 +189,12 @@ def shorter_than_minimum_duration(self):
         shorter than the MINIMUM_DURATION in minutes.
         """
         if self.duration:
```
Comment (nataliemona) [inline functions/zoom-downloader.py:191]: here's the bug, I thought tm_min would give total minutes

File: tests/test_downloader.py [hunk 2]
```diff
@@ -228,6 +237,7 @@ def test_duration(mocker):
 def test_shorter_than_minimum_duration():
     cases = [
         ({"duration": "02:45:01"}, False),
```
Comment (nataliemona) [inline tests/test_downloader.py:239]: and this test case had tm_min = 45 so it missed a corner case where tm_min = 1 or tm_min = 0
